src/other/Table.js

```diff
@@ -235,13 +235,51 @@
         }
         var idx = this._columns.indexOf(column);
 
-        this._data.sort(function (l, r) {
-            if (l[idx] === r[idx]) {
-                return 0;
-            } else if (typeof (r[idx]) === "undefined" || l[idx] > r[idx]) {
-                return context._currentSortOrder;
+        // http://www.davekoelle.com/files/alphanum.js
+        // http://stackoverflow.com/questions/2802341/javascript-natural-sort-of-alphanumerical-strings
+
+        var caseSensitive = true;
+        this._data.sort(function alphanum(a, b) {
+            function chunkify(t) {
+                var tz = [];
+                var x = 0, y = -1, n = 0, i, j;
+                while (i = (j = t.charAt(x++)).charCodeAt(0)) { // jslint: suggestions?
+                    var m = (i === 46 || (i >= 48 && i <= 57));
```

Comment:
It shouldn't affect it at all (except maybe running slightly faster?). "===" is just a strict comparison and should behave the same as == when comparing integers.